<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>国际象棋</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/chess.ico</normaloff>:/chess.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber_Timer">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Submit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>认输</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_check">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>将
军</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="action_setting"/>
   </widget>
   <widget class="QMenu" name="menu_network">
    <property name="title">
     <string>网络设置</string>
    </property>
    <addaction name="action_host"/>
    <addaction name="action_connect"/>
   </widget>
   <widget class="QMenu" name="menu_remain">
    <property name="title">
     <string>残局设置</string>
    </property>
    <addaction name="action_loaddefault"/>
    <addaction name="action_load"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
   </widget>
   <addaction name="menu_setting"/>
   <addaction name="menu_network"/>
   <addaction name="menu_remain"/>
  </widget>
  <action name="action_host">
   <property name="text">
    <string>建立主机</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="text">
    <string>连接主机</string>
   </property>
  </action>
  <action name="action_setting">
   <property name="text">
    <string>游戏设置</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>载入残局</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存残局</string>
   </property>
  </action>
  <action name="action_loaddefault">
   <property name="text">
    <string>默认棋局</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
